src/Stormpath/DataStore/DefaultDataStore.php

```diff
@@ -334,4 +339,13 @@ private function getQueryString(array $options) {
 
         return $query;
     }
+
+    public function getCacheManager() {
+        return $this->cacheManager;
+    }
+
+    public function getApiKey()
+    {
+        return $this->requestExecutor->getApiKey();
+    }
```

Comment:
Please make these new functions (getCacheManager and getApiKey) protected because we don't want to expose these to customers. We might change/remove them in the future and we don't want customers to asume that they can use them by their visibility. The getRequestExecutor method can be removed, I don't think we need it here.

Also, this doesn't necessarily needs to change now, but when I suggested the change of getting the api key from the data store I was thinking about moving the apiKey to the data store as an instance property and not have it in the request executor anymore (just pass it as a parameter or something).